packages/terra-slide-panel/src/terra-dev-site/test/slide-panel/SlidePanelSideToggle.test.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import SlidePanel from '../../../SlidePanel';
+import styles from './SlidePanelDocCommon.test.module.scss';
+
+const cx = classNames.bind(styles);
+
+class ButtonWrapper extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { count: 0 };
+    this.increaseCount = this.increaseCount.bind(this);
+  }
+
+  increaseCount() {
+    this.setState(prevState => ({ count: prevState.count + 1 }));
+  }
+
+  render() {
+    return (
+      <div id="panel-content" className={cx('panel-content')}>
+        <button id="focus-button" type="button" className={cx('button')} onClick={this.increaseCount}>Increase Count </button>
+        {this.state.count}
+      </div>
+    );
+  }
+};
+
+
+class SlidePanelSideDemo extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { panelIsOpen: false };
```

Comment:
Can we update this state value to something that represents toggling the panel alignment?

`panelHasStartPostion` maybe?

Reply:
[Fixed](https://github.com/cerner/terra-framework/pull/1071/commits/f13cab61cea812d92a6bcc163b0aa814be8ab3f5#diff-7e3ed4ed30d6886ce786592bdd93daf2R6)